### PR TITLE
[PW-6479] Merge Klarna Tests to main

### PR DESCRIPTION
**Pull request by cenkiravul**
## Tested scenarios

**Fixed issue**:

### CODE REVIEW

File: projects/magento/magentoCI.config.cjs [hunk 1]
```diff
@@ -37,7 +37,9 @@ const config = {
     actionTimeout: 0,
 
     /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: process.env.MAGENTO_BASE_URL,
+    // baseURL: process.env.MAGENTO_BASE_URL,
+    baseURL: "magento2.test.com",
```
Comment (michaelpaul) [inline projects/magento/magentoCI.config.cjs:41]:
```suggestion
    baseURL: "http://magento2.test.com",
```

File: projects/magento/magentoCI.config.cjs [hunk 2]
```diff
@@ -37,7 +37,9 @@ const config = {
     actionTimeout: 0,
 
     /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: process.env.MAGENTO_BASE_URL,
+    // baseURL: process.env.MAGENTO_BASE_URL,
+    baseURL: "http://magento2.test.com",
```
Comment (michaelpaul) [inline projects/magento/magentoCI.config.cjs:41]: Ops, we can't actually do this. It will break this repository's actions. :(

Comment (cenkiravul) [reply projects/magento/magentoCI.config.cjs:41]: I won't break the actions, since we use another config file for the repo itself.

Comment (wboereboom) [reply projects/magento/magentoCI.config.cjs:41]: Can we remove the commented code either way?

**Review by wboereboom — APPROVED** [review]

File: projects/magento/magentoCI.config.cjs (hunk 2 above)
Comment (cenkiravul) [reply projects/magento/magentoCI.config.cjs:40]: Done!

**Review by michaelpaul — APPROVED** [review]

**Pull request merged**